### DISCUSSION TITLE
大大，如何查看实时的dns解析日志

### DISCUSSION THREAD

**Discussion by p1ayer222** [kkkgo/PaoPaoDNS#99] [问答和帮助] [answered]
### 在提交之前，请确认

- [X] 我已经尝试搜索过Issue和看过参数说明，但没有找到相关问题。
- [X] 我正在使用最新的docker镜像版本（可以尝试`docker pull sliamb/paopaodns:latest`后重新创建容器，如果不确定的话可以尝试使用`sliamb/paopaodns:dev`）。

### 脚本自检日志

```txt
ALL TEST PASS
```

### 问题描述和复现步骤

Rt

**Comment by kkkgo** [comment, marked as answer]
如果你会编辑mosdns配置的话可以把level: error改成debug，然后添加summary。
更简单的方法是你在前面套一个adguardhome，可以网页点点点看请求日志。
如果你是想了解DNS结果来源的话我建议你使用ADDINFO功能。
